tests/scenarios/blog-widget.spec.js

```diff
@@ -0,0 +1,74 @@
+const server = require('../server');
+const steps = require('../steps/index');
+
+module.exports = Object.assign(
+  {
+    before: (client, done) => {
+      client.resizeWindow(1200, 800);
+
+      this._server = server.create();
+      this._server.start(done);
+    },
+
+    after: (client, done) => {
+      client.end(() => {
+        this._server.stop(done);
+      });
+    },
+  },
+  steps.main(),
+  steps.login(),
+  steps.switchLocale('es'),
+  steps.switchToDraftMode(),
+  steps.makeSubPage('Regression test'),
+  steps.createArticle('New Blog Article Title'),
+  {
+    'add a blog widget to the regression test page': (client) => {
+      const finishBtnSelector = '[data-apos-cancel]';
+      const blackoutSelector = '.apos-modal-blackout';
+      const mainBlockSelector = '.demo-main';
+      const addContentBtnSelector = `${mainBlockSelector} [data-apos-add-content]`;
+      const blogWidgetBtnSelector = `${mainBlockSelector} [data-apos-add-item=apostrophe-blog]`;
+      const modalDialogSelector = '.apos-modal.apos-modal-slideable';
+      const selectBlogTypeSelector = '.apos-field-select select';
+      const browseBtnSelector = '[data-apos-browse]';
+      const articleCheckboxSelector = '.apos-manage-table tr[data-piece] .apos-field-input-checkbox-indicator';
+      const busyLayerSelector = '.apos-global-busy.active';
+      const saveChoicesBtnSelector = '(//a[@data-apos-save])[2]';
```

Comment:
I think we can do it without xPath. All modal dialog has own particular class.

Reply:
leave